sig-and-wg/sig-core/proposals/test-folder-consolidation.md

```diff
@@ -0,0 +1,67 @@
+# Test folder consolidation
+
+## Introduction
+The [tests](https://github.com/kyma-project/kyma/tree/master/tests) folder of the kyma-project contains tests executed against a running Kyma cluster to assure integrity and functional correctness of the cluster with all modules installed.
+These are acceptance tests.
```

Comment:
```suggestion

```

Remove the line and merge it with the paragraph above.

Reply:
No, I don't like the idea. We are talking about the tests contained in that folder. I want to express that these tests are covering one test suite each and they act in the role of acceptance tests. That consideration should be the outcome of the paragraph for the reader.

Reply:
Ok, sure - let's leave it then.